net/shorewall6-lite/Makefile

```diff
@@ -28,7 +28,7 @@ include $(INCLUDE_DIR)/package.mk
 define Package/shorewall6-lite
     SECTION:=net
     CATEGORY:=Network
-    DEPENDS:=+ip +iptables6 +shorewall-core
+    DEPENDS:=+ip +iptables +shorewall-core
```

Comment:
Your fix (using iptables) looks strange. You probably should have "ip6tables" there, if the goal is to control ipv6 firewall.